### ISSUE TITLE
In wheels `bin/numba` is not executable

### ISSUE THREAD

**Issue by stuartarchibald** [numba/numba#5379]
## Reporting a bug

- [x] I am using the latest released version of Numba (most recent is visible in
 the change log (https://github.com/numba/numba/blob/master/CHANGE_LOG).
- [x] I have included below a minimal working reproducer (if you are unsure how
 to write one see http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports).

Example:
```
$ conda create -n envname python=3 numpy
Collecting package metadata (current_repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.7.10
  latest version: 4.8.2

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: <path>/anaconda3/envs/envname

  added / updated specs:
    - numpy
    - python=3


The following NEW packages will be INSTALLED:

  _libgcc_mutex      pkgs/main/linux-64::_libgcc_mutex-0.1-main
  blas               pkgs/main/linux-64::blas-1.0-mkl
  ca-certificates    pkgs/main/linux-64::ca-certificates-2020.1.1-0
  certifi            pkgs/main/linux-64::certifi-2019.11.28-py38_0
  intel-openmp       pkgs/main/linux-64::intel-openmp-2020.0-166
  ld_impl_linux-64   pkgs/main/linux-64::ld_impl_linux-64-2.33.1-h53a641e_7
  libedit            pkgs/main/linux-64::libedit-3.1.20181209-hc058e9b_0
  libffi             pkgs/main/linux-64::libffi-3.2.1-hd88cf55_4
  libgcc-ng          pkgs/main/linux-64::libgcc-ng-9.1.0-hdf63c60_0
  libgfortran-ng     pkgs/main/linux-64::libgfortran-ng-7.3.0-hdf63c60_0
  libstdcxx-ng       pkgs/main/linux-64::libstdcxx-ng-9.1.0-hdf63c60_0
  mkl                pkgs/main/linux-64::mkl-2020.0-166
  mkl-service        pkgs/main/linux-64::mkl-service-2.3.0-py38he904b0f_0
  mkl_fft            pkgs/main/linux-64::mkl_fft-1.0.15-py38ha843d7b_0
  mkl_random         pkgs/main/linux-64::mkl_random-1.1.0-py38h962f231_0
  ncurses            pkgs/main/linux-64::ncurses-6.2-he6710b0_0
  numpy              pkgs/main/linux-64::numpy-1.18.1-py38h4f9e942_0
  numpy-base         pkgs/main/linux-64::numpy-base-1.18.1-py38hde5b4d6_1
  openssl            pkgs/main/linux-64::openssl-1.1.1d-h7b6447c_4
  pip                pkgs/main/linux-64::pip-20.0.2-py38_1
  python             pkgs/main/linux-64::python-3.8.1-h0371630_1
  readline           pkgs/main/linux-64::readline-7.0-h7b6447c_5
  setuptools         pkgs/main/linux-64::setuptools-46.0.0-py38_0
  six                pkgs/main/linux-64::six-1.14.0-py38_0
  sqlite             pkgs/main/linux-64::sqlite-3.31.1-h7b6447c_0
  tk                 pkgs/main/linux-64::tk-8.6.8-hbc83047_0
  wheel              pkgs/main/linux-64::wheel-0.34.2-py38_0
  xz                 pkgs/main/linux-64::xz-5.2.4-h14c3975_4
  zlib               pkgs/main/linux-64::zlib-1.2.11-h7b6447c_3


Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate envname
#
# To deactivate an active environment, use
#
#     $ conda deactivate

(base)
$ conda activate envname

(envname)
$ pip install numba
Collecting numba
  Downloading numba-0.48.0-cp38-cp38-manylinux1_x86_64.whl (2.5 MB)
     |████████████████████████████████| 2.5 MB 2.1 MB/s
Requirement already satisfied: setuptools in <path>/anaconda3/envs/envname/lib/python3.8/site-packages (from numba) (46.0.0.post20200309)
Requirement already satisfied: numpy>=1.15 in <path>/anaconda3/envs/envname/lib/python3.8/site-packages (from numba) (1.18.1)
Collecting llvmlite<0.32.0,>=0.31.0dev0
  Using cached llvmlite-0.31.0-cp38-cp38-manylinux1_x86_64.whl (20.2 MB)
Installing collected packages: llvmlite, numba
Successfully installed llvmlite-0.31.0 numba-0.48.0
(envname)

$ numba -s
bash: <path>/anaconda3/envs/envname/bin/numba: Permission denied

$ ls -la <path>/anaconda3/envs/envname/bin/numba
-rw-rw-r--. 1 user user 226 Mar 11 23:22 <path>/anaconda3/envs/envname/bin/numba
```

**Comment by sklam** [comment]
This should now be fixed in the upcoming 0.49.0rc1 wheels.

**Comment by stuartarchibald** [comment]
Packages in your channel work:
```
$ pip list
Package     Version            
----------- -------------------
certifi     2019.11.28         
llvmlite    0.31.0             
mkl-fft     1.0.15             
mkl-random  1.1.0              
mkl-service 2.3.0              
numba       0.49.0rc1          
numpy       1.18.1             
pip         20.0.2             
setuptools  46.1.3.post20200330
six         1.14.0             
wheel       0.34.2             

$ numba -s|head -7
System info:
--------------------------------------------------------------------------------
__Time Stamp__
2020-04-02 14:23:24.692721

__Hardware Information__
Machine                                       : x86_64

```
